File: engine/scripts/gen_chapter_tex.py
#!/usr/bin/env python

import sys

the_chapter = sys.argv[1]

front = """\\input{preamble.tex}

\\begin{document}

\\renewcommand{\\bibname}{References}

\\mainmatter"""

back = """\n
\\backmatter

\\end{document}
"""

print front + "\n\n\\include{" + the_chapter + "-frag}\n\n" + back

File: engine/scripts/gen_chapter_tex.R
#!/usr/bin/env Rscript

the_chapter <- commandArgs(trailingOnly = TRUE)[1]

front <- "\\input{preamble.tex}\n
\\begin{document}
\\renewcommand{\\bibname}{References}
\\mainmatter"

back <- "\\backmatter
\\end{document}"

cat(front, "\n\n\\include{", the_chapter, "-frag}\n\n", back, "\n", sep = "")
